homeassistant/components/prometheus/__init__.py

```diff
@@ -807,6 +817,34 @@ def _handle_update(self, state: State) -> None:
         value = self.state_as_number(state)
         metric.labels(**self._labels(state)).set(value)
 
+    def _handle_alarm_control_panel(self, state: State) -> None:
+        current_mode = state.state
+
+        available_states = [
```

Comment:
Instead of hardcoding this list here, could we use `VALID_STATES` from `alarm_control_panel`?

https://github.com/home-assistant/core/blob/6406065e1f43cc92635619eb44d453eeafa03067/homeassistant/components/alarm_control_panel/reproduce_state.py#L33-L41

Reply:
I'm using `VALID_STATES` now. I had to add three states that were missing from this.